Src/AutoFixtureUnitTest/DataAnnotations/StringLengthAttributeRelayTest.cs

```diff
@@ -112,5 +112,29 @@ public void CreateWithConstrainedStringRequestReturnsCorrectResult(int maximum)
             Assert.Equal(expectedResult, result);
             // Teardown
         }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        public void CreateWithStringRequestConstrainedbyMinimunLengthReturnsCorrectResult(int maximum)

```

Comment:
That [other](https://github.com/sergeyshushlyapin/AutoFixture/blob/4d526237d012bbfea42fe9e4e9d75db92d8dabc9/Src/AutoFixtureUnitTest/DataAnnotations/StringLengthAttributeRelayTest.cs#L94) one exercises just the `MaximumLength`, while this one does both. I think it wouldn't hurt to keep both. I'd be nice, IMO, if we can keep [test suites append-only](http://blog.ploeh.dk/2013/04/02/why-trust-tests/).